analysis/question_2/top_suicide_rate.py

```diff
@@ -0,0 +1,15 @@
+import os
+import pandas as pd
+
```

Comment:
I would like to suggest to include introductory comment about the overall analysis at the beginning of the code. Example - objective

Reply:
Agree with HninPwint